src/phoenix/trace/schemas.py

```diff
@@ -4,6 +4,8 @@
 from typing import Any, Dict, List, Optional, Union
 from uuid import UUID
 
+from phoenix.trace.semantic_conventions import EXCEPTION_MESSAGE
```

Comment:
I think it's not super clear that this is specific just to events and that the namespace is different?

Reply:
I see, got it